Library/Homebrew/formula_installer.rb

```diff
@@ -1145,19 +1145,19 @@ def pour
     keg = Keg.new(formula.prefix)
     Tab.clear_cache
 
+    tabfile = keg/Tab::FILENAME
+    bottle_json_path = formula.local_bottle_path&.sub(/\.tar\.gz$/, ".json")
```

Comment:
I think this won't always work (or at least I've seen it fail)? I recommend using `Bottle::Filename#json` for this (and probably the case in `utils/bottles` too.

Reply:
I need to at least extract the tag from this path though. Is that safe to do?

Reply:
Hmm, yeh. I guess could extract the `tab =` part into a function elsewhere? Maybe overkill though, feel free to punt.